scripts/reporting/one_line_redcap.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -i
```

Comment:
Since this file just creates a dashboard, it should go to `dashboards/`.

Reply:
Moved to correct folder.